src/App/Handler/MaintenanceOverviewHandler.php

```diff
@@ -15,6 +15,23 @@ class MaintenanceOverviewHandler implements RequestHandlerInterface
     public const string CUSTOM_PROPERTIES_FILE      = 'maintenance-status.json';
     public const string CUSTOM_PROPERTIES_DIRECTORY = '/public/share';
 
+    public const array MVC_COMPONENTS = [
+        'laminas-mvc',
+        'laminas-modulemanager',
```

Comment:
module manager is not a mvc component

Reply:
@froschdesign and @Xerkus let's produce a list of all MVC -related components , to correct this filter

Reply:
But listed under MVC for years: https://docs.laminas.dev/mvc/

Reply:
I will update the documentation overviews and move laminas-modulemanager and laminas-psr7bridge to components.

Reply:
Ok, give me 30 minutes :-)